include/ginkgo/core/base/executor.hpp

```diff
@@ -1062,6 +1111,8 @@ class CudaExecutor : public detail::ExecutorBase<CudaExecutor>,
     friend class detail::ExecutorBase<CudaExecutor>;
 
 public:
+    using cuda_exec_info = exec_info;
```

Comment:
Why not use `exec_info` directly? That seems like much simpler, and shorter as well.

Reply:
I wanted some leeway, in case we decide to add some executor specific info, we can create a new struct `cuda_exec_info` and still not break the interface.